### DISCUSSION TITLE
How do I see who left an emoji reaction?

### DISCUSSION THREAD

**Discussion by lucasmbrown-usds** [GitbookIO/community#467] [1. How to]
Comments on change requests allow people to emoji-react to them. In Slack and GitHub and other products, I can see who left that emoji reaction.

In GitBook, I don't seem to be able to see who it was that emoji reacted to the comment. I mouse over it and nothing shows up. 

Is this a bug or am I not properly using the feature? 

![image](https://github.com/GitbookIO/community/assets/25443529/a4093083-4f1e-46ba-a3b9-37bbfaedcaae)

**Comment by addisonschultz** [comment]
Hey @lucasmbrown-usds, great catch! I think this is a bit broken right now. 

Let me make a note for our team to look into!
